src/app/certificates/certificates.component.ts

```diff
@@ -8,11 +23,27 @@ import { Certificate, CertificatesService } from '../shared/services/certificate
   styleUrls: ['./certificates.component.scss']
 })
 export class CertificatesComponent implements OnInit {
-  certificates: Certificate [];
+  certificates: Certificate [] = MyCertificate;
+  currentCertificate: Certificate;
 
-  constructor( private certificatesService: CertificatesService ) { }
+  constructor() {
+    this.currentCertificate = this.certificates[0];
+  }
 
   ngOnInit() {
-    this.certificates = this.certificatesService.getAll();
+    // this.certificates = this.certificatesService.getAll();
+  }
+
+  changeCertificate(param) {
```

Comment:
@Yuriy-27 `param` is not clear, what it is? please change the better name and specify type